dev-tools/cmd/asset/asset.go

```diff
@@ -101,6 +101,8 @@ func main() {
 	if name == "" {
 		name = file
 	}
+	// Capitalise first letter
+	name = strings.Title(name)
 	asset.Template.Execute(&buf, asset.Data{
```

Comment:
I think it would be useful to have a comment in the generated source that says where the data was sourced from (even if that file itself is a generated).

```
// AssetBeat returns asset data. (<- NOTE I added a period to end this sentence.)
// This is the base64 encoded gzipped contents of {{ file_path }}.
func AssetBeat() string {
```